### PR TITLE
Feature/address view area update

### PR DESCRIPTION
**Pull request by Dremars**
Close-by services tab for address view.
Link component for unified text link stylings

### CODE REVIEW

**Comment by Dremars** [comment]
Waiting for missing translations.

**Comment by Dremars** [comment]
Need to change which markers are drawn on map based on selected tab

**Comment by Dremars** [comment]
Map drawing updated in 7bf7065

File: src/components/ListItems/DivisionItem/DivisionItem.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    : ''} 
+  `;
+
+  return (
+    <>
+      <ListItem
+        component="li"
+        className={classes.listItem}
+      >
+        <SMLink
```
Comment (mikkoisa) [inline src/components/ListItems/DivisionItem/DivisionItem.js:53]: Could SMLink-component be replaced by Mui ButtonBase-component? ButtonBase with styles: flexDirection: 'column' and alignItems: 'initial' seems to be working identically.

Comment (Dremars) [reply src/components/ListItems/DivisionItem/DivisionItem.js]: Changed in fecf6c7

**Comment by Dremars** [comment]
> The font size and style look a little different compared to the design layout. Also looks like the names should be of type "Neuvola" instead of "Neuvola-alue" .
> Näyttökuva 2020-4-29 kello 13 20 07

We talked about names earlier in meetings and decided it's fine with area text. Style can't be exactly same since original design didn't give direction for links.

**Comment by mikkoisa** [comment]
Ok, looks good!

**Pull request merged**